app/components/trainees/confirmation/schools/view.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module Trainees
+  module Confirmation
+    module Schools
+      class View < GovukComponent::Base
```

Comment:
There's a bit of overlap here with this component https://github.com/DFE-Digital/register-trainee-teachers/blob/master/app/components/trainees/school_details/view.rb. I wonder if some of the methods could be extracted and re-used across the two instead of being duplicated. Could have a schools_helper.rb module which is included into both?

Reply:
i think we already have a schools helper actually!